astropy/units/quantity_helper/function_helpers.py

```diff
@@ -118,7 +128,10 @@
     np.save, np.savez, np.savetxt, np.savez_compressed,
     # Polynomials
     np.poly, np.polyadd, np.polyder, np.polydiv, np.polyfit, np.polyint,
-    np.polymul, np.polysub, np.polyval, np.roots, np.vander}
+    np.polymul, np.polysub, np.polyval, np.roots, np.vander,
+    # record array functions
+    rfn.rec_append_fields, rfn.rec_drop_fields, rfn.rec_join,
```

Comment:
Why are these added to `ignored` while the others are in `TOSUPPORT`?

Reply:
Because they work on record arrays, which are deprecated.
Does Quantity even work with a record array?

Reply:
OK, that makes sense, and I now see that I could have read that from the comment. Maybe make it a bit more explicit? E.g., "functions taking record arrays (which are deprecated)"